src/main/java/frc/robot/Robot.java

```diff
@@ -68,9 +69,10 @@ public Robot()
   private void initialize()
   {
     drivetrain.stopMotors();
+    SmartDashboard.putString("Drive Speed", "HIGH");
 
     intake.stop();
-    intake.retract();
+    intake.deploy(); // Deploy default for humber
```

Comment:
Does this mean that the solenoid channels are backwards?  Or can we just flip the function names to make them consistent?

Reply:
No, it doesn't mean that. Having retract as the default meant that the intake flip button had to be pressed every time the robot turned on. Since the intake piston isn't installed yet, there's no point in doing that, so I made deploy the default to avoid confusion. We can make retract the default again once the piston is added.

Reply:
Ah yes you're right, since the intake will not spin unless deployed.  Makes sense.